ui/src/components/CsvSourceMapping.vue

```diff
@@ -18,9 +18,11 @@
               Create table from selected columns
             </o-button>
             <o-dropdown position="bottom-left" class="has-text-weight-normal" aria-role="list">
-              <button class="button is-text is-small" slot="trigger">
-                <o-icon icon="ellipsis-h" />
-              </button>
+              <template #trigger>
+                <button class="button is-text is-small">
+                  <o-icon icon="ellipsis-h" />
+                </button>
+              </template>
```

Comment:
I'm curious about this

Reply:
The `slot` attribute has been deprecated https://v2.vuejs.org/v2/api/#slot-deprecated